### PR TITLE
Adding CSS to Failure Instance Panel

### PR DESCRIPTION
**Pull request by annikakouhia**
#### Description of changes

CSS fixes to match Figma. Changes occur on the Failure Instance Panel as well as the list of Failure Instances that grows on the main page as you add new Failure Instances.

To Note:
-- Duplicated JS functions to set the style of two TextFields (Comment in FIPC and CSS Selector in FIPD)
-- Still working with Fer and Chelsea to decide if we want to change the wording on the Validate Selector button
-- Path Snippet Store changes are coming in separate PR
-- Changes on the screen where you click to open the panel (with the list of added failure instances) is NOT behind feature flag (approved by Fer as long as I implemented her thoughts correctly)
-- Footer should have a bar above it, but that's for a Panel and we use a GenericPanel (may be another Fabric version issue as well)
-- Can't add placeholder text to TextFields until we update Fabric Version

#### Pull request checklist

- [X] Addresses an existing issue: PR for Feature #1555720
- [X] Added relevant unit test for your changes. (`yarn test`)
- [X] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [X] Ran precheckin (`yarn precheckin`)
- [X] (UI changes only) Added screenshots/GIFs to description above
- [X] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

**Comment by annikakouhia** [comment]
@waabid I've simplified multiple line styles down to a single line as you suggested (i.e. putting all padding values on one line). I also talked to Chelsea and got colors to be ones from our Colors file. Finally, I put in JS functions to safely set the styling of subcomponents of TextFields as suggested in Fabric documentation. Let me know if you think any of these issues are still actionable!

**Comment by annikakouhia** [comment]
@Shobhit1 
1. Chelsea had accidentally used a few new colors in the Figma, but we spoke and got it figured out so they're all now cleared by Chelsea and from the Colors.scss file!
2. In the Assessment-instance-details-column file, the space after {this.props.headerText} was intentional (I put in a CSS attribute to make sure it wouldn't collapse) so we would have a space between the header ("Comment:") and the value afterward. I could also add the space to the header itself? Making it "Comment: " instead of "Comment:". What do you think?
3. Whoops the other space you caught (before "Code..." in the Failure-Instance-Panel-Details file) was unintentional. I've taken it out.
4. In the Path-snippet-action-message-creator file I added a new parameter first in the method because I originally was thinking it would be required, but the (now second) parameter would be optional so I wanted to order them. I think they'll both be mandatory so less important now :)

Let me know if I should address these changes in any other way! Thank you!

File: src/DetailsView/components/failure-instance-panel-control.tsx [hunk 1]
```diff
@@ -135,29 +137,37 @@ export class FailureInstancePanelControl extends React.Component<FailureInstance
...
+    private getActionCancelButtons = (): JSX.Element => {
+        return (
+            <div className="footer">
                 <ActionAndCancelButtonsComponent
                     isHidden={false}
                     primaryButtonDisabled={
                         this.state.currentInstance.failureDescription === null && this.state.currentInstance.path === null
                     }
-                    primaryButtonText={this.props.actionType === CapturedInstanceActionType.CREATE ? 'Add' : 'Save'}
+                    primaryButtonText={this.props.actionType === CapturedInstanceActionType.CREATE ? 'Add failed instance' : 'Save'}
```
Comment (waabid) [inline src/DetailsView/components/failure-instance-panel-control.tsx]: I've noticed you're checking for this "Create" action type twice; you could isolate this, at least visually, by putting them in local variables (that you reference here) in an if statement before the return statement.

Comment (annikakouhia) [reply src/DetailsView/components/failure-instance-panel-control.tsx]: Got it! Created a new Boolean variable called createMode

Comment (annikakouhia) [reply src/DetailsView/components/failure-instance-panel-control.tsx]: Boolean in and tests are passing! Thank you!

**Pull request merged**